pandas/core/categorical.py

```diff
@@ -1862,8 +1990,11 @@ def convert_categorical(x):
     if get_dtype_kinds(to_concat) - set(['object', 'category']):
         # convert to object type and perform a regular concat
         from pandas.core.common import _concat_compat
-        return _concat_compat([np.array(x, copy=False, dtype=object)
-                               for x in to_concat], axis=0)
```

Comment:
Definitely the best solution is to twiddle some knobs until wes is happy (that way he'll be perpetually happy!) rather than patching things after the fact. Perhaps these commented examples could be good yapf config test cases for us??